services/movie.py

```diff
@@ -0,0 +1,45 @@
+from django.db.models import Q, QuerySet
+
+from db.models import Genre, Actor, Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    movies = Movie.objects.all()
+
+    if genres_ids and actors_ids:
```

Comment:
you don't need this condition two `if` statements should be enough for this function to produce correct result